Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoISE.yaml

```diff
@@ -0,0 +1,129 @@
+        hostname_has_any:dynamic=dynamic([]), 
+        dvcaction:dynamic=dynamic([]), 
+        eventresult:string='*', 
+        disabled:bool=false){
+        let src_or_any=set_union(srcipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+        let dst_or_any=set_union(dstipaddr_has_any_prefix, ipaddr_has_any_prefix);
+        let ip_any = set_union(srcipaddr_has_any_prefix, dstipaddr_has_any_prefix, ipaddr_has_any_prefix); 
+        Syslog
+        | where not(disabled)
+        | where ProcessName has_any ("CISE", "CSCO")
+        | parse kind = regex SyslogMessage with @"\d{10}\s" EventOriginalType @"\s(NOTICE|INFO|WARN|WARNING|ERROR|FATAL|DEBUG)"
+        //***************************** <Prefiltering> **************************
+        | where (isnull(starttime) or TimeGenerated>=starttime) 
+            and (isnull(endtime) or TimeGenerated<=endtime) 
+            and not(disabled)
+            and (array_length(hostname_has_any) == 0)
+            and (array_length(dvcaction) == 0)
+            and (EventOriginalType in ("25023","25024","25025","25026","25027","25028","25029","25030","25101","25102","60188","60234","60235","60236","61010","61011","61025","61026","60509","61231","61232","89003","24000","24001","24019","24030","24400","24401","24428","24429","24850","24851","34120","34147","34148","34149","34150","34159","34160","34161","34162","34163","34164","90062","90063","90066","90078","91082","11317","5405","5406"))
+            and (array_length(ip_any)==0 or has_any_ipv4_prefix(SyslogMessage,ip_any)) 
+            and (isnull(dstportnumber) or SyslogMessage has (strcat('DestinationPort=', tostring(dstportnumber))))
+        | lookup EventName on EventOriginalType
+        | where (eventresult == "*" or eventresult == EventResult)
+        //**************************** </Prefilterring> **************************
+        | parse-kv SyslogMessage as (FailureReason:string, NetworkDeviceName:string, DestinationIPAddress:string, DestinationPort:int, ['Remote-Address']:string, ['Device IP Address']:string, ['User-Name']:string, UserName:string, User:string, ['Device Port']:int, Protocol:string, ['Caller-Station-ID']:string, ['Called-Station-ID']:string) with (pair_delimiter=',', kv_delimiter='=')
+      | project-rename DstIpAddr=DestinationIPAddress
+                      , DstPortNumber=DestinationPort
+                      , SrcPortNumber=['Device Port']
+                      , NetworkApplicationProtocol=Protocol
+      | invoke _ASIM_ResolveSrcFQDN("['Caller-Station-ID']")
+      | extend 
+              EventVendor = "Cisco"
+              , EventProduct = "ISE"
+              , EventProductVersion = "3.2"
+              , EventCount = int(1)
+              , EventSchema = "NetworkSession"
+              , EventSchemaVersion = "0.2.6"
+              , EventStartTime = coalesce(EventTime, TimeGenerated)
+              , EventEndTime = coalesce(EventTime, TimeGenerated)
+              , EventType = "NetworkSession"
```

Comment:
Can we check if EventSubType is applicable in some of these?

Reply:
Added EventSubType to lookup table